ovs/dal/hybrids/vdisk.py

```diff
@@ -296,3 +302,15 @@ def calculate_delta(key, dynamic, current_stats):
                 else:
                     current_stats['{0}_ps'.format(key)] = max(0, (current_stats[key] - previous_stats[key]) / delta)
         volatile.set(prev_key, current_stats, dynamic.timeout * 10)
+
+    def reload_clients(self, client):
```

Comment:
Agree, ```reload_client``` might have been more appropriate, but at least, it's consistent with ```vPool```. Feel free to rename in a future change, but I won't make a new PR just for this renaming.